internal/painter/gl/gl_core.go

```diff
@@ -187,7 +187,7 @@ func (p *glPainter) glCreateBuffer(points []float32) Buffer {
 
 	texCoordAttrib := uint32(gl.GetAttribLocation(uint32(p.program), gl.Str("vertTexCoord\x00")))
 	gl.EnableVertexAttribArray(texCoordAttrib)
-	gl.VertexAttribPointer(texCoordAttrib, 2, gl.FLOAT, false, 5*4, gl.PtrOffset(3*4))
+	gl.VertexAttribPointer(texCoordAttrib, 2, gl.FLOAT, false, 5*4, gl.PtrOffset(12))
```

Comment:
not sure this is part of this PR
why change 3 * 4 to 12 but not 5 * 4 to 20?
what are these magic numbers?

Reply:
3*4 was creating a compile error due to it thinking this was bad pointer arithmatic, so I only changed what was needed.
The offset is, iirc, the 3 vertexes in to a 4 channel colour. Honestly I can't remember - it requires special concentration to get GL code to make sense ;)